### ISSUE TITLE
[Search] Search broken and returns 500 Internal Server Error

### ISSUE THREAD

**Issue by 1000miles** [sozialhelden/wheelmap#674]
# Background

When using the search function on wheelmap.org it returns a `500 Internal Server Error`. In the backend it outputs `301 Moved Permanently` which indicates that it might be related to the https move by openstreetmap from 2018-05-07 on. 

```shell

Completed 500 Internal Server Error in 185ms (ActiveRecord: 11.3ms)

JSON::ParserError (795: unexpected token at '<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://nominatim.openstreetmap.org/search?accept-language=de&amp;dedupe=1&amp;email=info%40wheelmap.org&amp;format=json&amp;limit=10&amp;q=asquera">here</a>.</p>
<hr>
<address>Apache/2.4.18 (Ubuntu) Server at nominatim.openstreetmap.org Port 80</address>
</body></html>
'):
  app/controllers/search_controller.rb:38:in `block (2 levels) in index'
  app/controllers/search_controller.rb:22:in `index'
  lib/rack_i18n_locale_switcher.rb:29:in `call'
```

# Story

Affected links (staging and production):
https://staging.wheelmap.org/search?q=something
https://wheelmap.org/search?q=something

As a user
When I type in a search term in the search input field 
I want to get a result based on my search term

**Comment by 1000miles** [comment]
@Svenyo cc @holgerd @anngreenberg 

This error is fixed by PR #673. Please close after successful testing.

### Production, 2018-05-16
![screen shot 2018-05-16 at 10 56 13](https://user-images.githubusercontent.com/4077916/40107184-dcf205dc-58f7-11e8-9ae2-cd962333eb65.jpg)

**Comment by holgerd** [comment]
yay! :)
